_attributes/common-attributes.adoc

```diff
@@ -39,6 +39,8 @@ endif::[]
 :oadp-full: OpenShift API for Data Protection
 :oadp-short: OADP
 :oadp-version: 1.4.0
+:oadp-version-1-3: 1.3.3
+:oadp-version-1-4: 1.4.0
```

Comment:
@anarnold97 the latest version validated for OADP 1.4 is 1.4.1. I suppose this should be set to `oadp-version-1-4: 1.4.1`. Please correct me if I'm wrong here. Thanks.

Reply:
@sonia-garudi - updated. if you are happy please add a LGTM

Thanks